src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -106,9 +106,53 @@ export class TelemetryComponent extends CdForm implements OnInit {
     return JSON.stringify(report, this.replacer, 2);
   }
 
+  private formatReport() {
+    let copy = {};
+    copy = JSON.parse(JSON.stringify(this.report));
```

Comment:
This seems to be the recommended way to perform deep-copy with JS objects, but when a reviewer stumbles upon this line they'll likely scratch their heads... Why not using Lodash [`_.cloneDeep()`](https://lodash.com/docs/4.17.15#cloneDeep)?

Reply:
Thanks for telling me about this, I'll use it in the future. And thanks again so much for taking a look at this!